docs/standard/commandline/snippets/get-started-tutorial/csharp/Stage1/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "scl": {
```

Comment:
it's worth noting that as of 7.0.100, the SDK will be able to use these `commandLineArgs` as well with `dotnet run` if you specify the `--launch-profile` argument. So `dotnet run --launch-profile scl` would allow for a consistent experience in VS/VSMac/CLI

Reply:
Added to the how-to-test instructions in the tutorial.